types/state.go

```diff
@@ -40,11 +41,9 @@ type State struct {
 	// DAHeight identifies DA block containing the latest applied Rollkit block.
 	DAHeight uint64
 
-	// In the MVP implementation, there will be only one Validator
-	NextValidators              *types.ValidatorSet
-	Validators                  *types.ValidatorSet
-	LastValidators              *types.ValidatorSet
-	LastHeightValidatorsChanged uint64
+	// For vA centralized sequencer, replace validator complexity with a sequencer pubkey.
+	// We could just scrape it out of the genesis, but storing it in the State is future-proofing for a key-rotation feature.
+	//Sequencer cmtcrypto.PubKey
```

Comment:
If you're following the comment:

`validator complexity with a sequencer pubkey`, then shouldn't `//Sequencer cmtcrypto.PubKey` be uncommented?

Reply:
we deleted it. just going to delete the comment as well.